core/modules/dashboard/includes/block.news.inc

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * @file
+ * Dashboard block displaying BackdropCMS.org news items.
+ */
+
+class DashboardNewsBlock extends Block {
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct($plugin_name, array $data) {
+    parent::__construct($plugin_name, $data);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Backdrop News');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getAdminTitle() {
+    return t('Backdrop News');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getAdminPreview() {
+    return t('Displays a feed of news from BackdropCMS.org.');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getContent() {
+    $feed = backdrop_http_request('https://backdropcms.org/api/backdrop-news');
+    $news = json_decode($feed->data);
+    $links = array();
+
+    if ($feed->code === '200') {
+      foreach ($news->nodes as $key => $node) {
+        $links[] = l(
+          $node->node->title,
+          $node->node->field_link
+        ) . ' - ' . $node->node->field_tagline;
+      }
+    }
+
+    $build = array(
+      '#theme' => 'dashboard_panel__news',
+    );
+    if (empty($links)) {
+      $build['list'] = array(
+        '#type' => 'markup',
+        '#markup' => t('No news at this time.'),
+      );
+    }
+    else {
+      $build['list'] = array(
+        '#theme' => 'item_list',
+        '#items' => $links,
+      );
+    }
+
+    return $build;
+  }
+  /**
+   * {@inheritdoc}
+   */
+  public function form(&$form, &$form_state) {
```

Comment:
Could we just not override the form, and leave no message?

Reply:
we can; should we?

Reply:
This would be a new pattern as I dont recall blocks without form input elements having markup. Usually theyre just blank, like the Local Items block or the Powered By block. Plus this block may be used anywhere, not just on the dashboard, so I think the message might be redundant.

Reply:
I've never noticed that about the `Powered by` block, but honestly I don't like it.  It seems confusing, i.e. I wish that one had some message as well saying what the purpose of the block is, why I can't edit it, and telling me that I can remove it.

I know I can remove it but does everyone?  Seems a bad UX to me.

Reply:
I agree. I do find it bad UX to see "blank" blocks; so I like having a short description in such cases.